README.md

```diff
@@ -508,6 +508,12 @@ Ahc\Cli\Output\Color::style('error', [
 ]);
 ```
 
+#### Disable colors
+
+```php
+Ahc\Cli\Output\Color::$enabled = false;
```

Comment:
in addition to manual control like this, there can be env vars like LS_COLOR/S, NO_COLOR/S that you may want to check together while at it :)

Reply:
LS_COLOR seemed to be specifically for the `ls` command but I like the idea of checking for NO_COLOR as described here: https://no-color.org/

So it'll now disable colors automatically of NO_COLOR environment variable is set to truthy.